connect-examples/server-library-example/client/src/main/scala/org/example/CustomTable.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.example
+
+import com.google.protobuf.Any
+import org.apache.spark.connect.proto.Command
+import org.example.proto
+import org.example.proto.CreateTable.Column.{DataType => ProtoDataType}
+import org.apache.spark.sql.{functions, Column, DataFrame, Dataset, Row, SparkSession}
```

Comment:
Apache Spark has an ordering rule for `import` statement.

Reply:
Fixed!